### ISSUE TITLE
Project needs to be deleted

### ISSUE THREAD

**Issue by Yudianzah** [notionparallax/ShadowWolf#826]
I need the project 'Yeshivah Beth Rivkah Colleges Masterplan' (M1304004 + M1307005) deleted from P&P as there is an individual project sheets on both the project numbers.

**Comment by Yudianzah** [comment]
@hinds89

**Comment by hinds89** [comment]
Note:
The Primary School tagged key brief elements from this project had already been copied across to project M1304004 (Yeshivah Beth Rivkah Colleges Masterplan), I tagged them as "Primary School".  If the primary school has its own project number then these should be moved there (but have just left them for now till the Yeshiva projects are interviewed again).
The Trade Technical Centre key brief elements had already been copied accross to project M1304004 (Yeshivah Beth Rivkah Colleges Masterplan) but I deleted them from there and moved to project M1310004 (Yeshivah Beth Rivkah - Trade Training Centre).
The rest of the info from this project looked like it had already been copied to the M1304004 project.
PROJECT DELETED
